content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,337 @@
+default, there are a collection of quotation and order settings that can be customized and
+configured, if needed.
+
+To access these setting options, navigate to :menuselection:`Sales app --> Configuration -->
+Settings`, and scroll to the :guilabel:`Quotations & Orders` section.
+
+.. image:: create_quotations/quotations-orders-section.png
+   :align: center
+   :alt: The Quotations and Orders section on the Odoo Sales app Settings page.
+
+- :guilabel:`Quotation Templates`: create unique quotation templates featuring standardized offers
+  with default products, which can be easily accessed from a quotation form. When this checkbox is
+  ticked, an additional field, :guilabel:`Default Template`, appears, along with a link to the
+  :guilabel:`Quotation Templates` page. To learn more, refer to the :doc:`Quotation templates
+  <quote_template>` documentation.
+- :guilabel:`Online Signature`: request an online signature to confirm orders. To learn more, refer
+  to the :doc:`Online payment order confirmation <get_signature_to_validate>` documentation.
+- :guilabel:`Online Payment`: request an online prepayment from customers to confirm orders. Request
+  a full or partial payment (via down payment). When this checkbox is ticked, an additional field,
+  :guilabel:`Prepayment amount (%)`, appears. There is also a link to the :guilabel:`Payment
+  Providers` page. To learn more, refer to the :doc:`Online payment order confirmation
+  <get_paid_to_validate>` documentation.
+- :guilabel:`Default Quotation Validity`: determine an amount (in :guilabel:`days`) that quotations
+  can remain valid for, by default.
+- :guilabel:`Default Recurrence`: select a default period from the drop-down menu to use as a
+  recurrence period for a new quotation.
+- :guilabel:`Sale Warnings`: get warning messages about orders that include specific products or
+  customers.
+- :guilabel:`PDF Quote builder`: customize the look of quotations with unique header pages, product
+  descriptions, footer pages, and more. To learn more, refer to the :doc:`PDF quote builder
+  <pdf_quote_builder>` documentation.
+- :guilabel:`Lock Confirmed Sales`: ensure that no further edits can be made to confirmed orders.
+- :guilabel:`Pro-Forma Invoice`: send pro-forma invoices to customers. To learn more, refer to the
+  :doc:`Pro-forma invoices <../invoicing/proforma>` documentation.
+
+To activate any of these settings, tick the checkbox beside the desired option(s). Then, click
+:guilabel:`Save`.
+
+Quotations dashboard
+====================
+
+To access the *Quotations* dashboard, which is the default dashboard of the *Sales* app, simply
+click the :menuselection:`Sales app` from the main Odoo dashbord.
+
+.. tip::
+   If already in the *Sales* application, the :guilabel:`Quotations` dashboard can be accessed by
+   navigating to :menuselection:`Orders --> Quotations` from the header menu in the *Sales*
+   application.
```

Comment:
Sentence too long to say a simple/obvious thing.
The tip isn't necessary.
Why 11 lines for such a basic navigational instruction?